testsuite/tests/ui/mail_and_messages/test_automatic_mails.py

```diff
@@ -2,14 +2,13 @@
 import re
 import pytest
 
-
```

Comment:
I know that this does not matter, anyway this is unnecessary change. (is it manual or pycharm?)

Reply:
black did it due to moving the test from this file